app/routers/quizzes.py

```diff
@@ -7,8 +7,9 @@
 router = APIRouter(prefix="/quiz", tags=["Quiz"])
 
 
-@router.post("/", response_model=QuizResponse)
+@router.post("/")
```

Comment:
Also, @deepansh96 can comment further, but instead of removing the response model, why don't we change the schema of the model.

Reply:
Oh definitely
We should never remove the response model for a request

@suryabulusu were you able to read the pydantic docs?
This response model thing is responsible for smartly typcasting the response, checking to see if the returned response follows a particular pattern or not.
This is a very helpful layer that reduces a lot of bugs.

Removing the response model just to make the response fit into something feels like a bandage applied over the issue.

You can just change the response model no?
then the outputted response will be formatted according to that ResponseModel class

Reply:
response model QuizResponse is already used in GET request `get_quiz`; so I kept it as is. I could've created a QuizIdResponse model but it Pritam said just returning `id` is fine

Reply:
Edit - read your message on discord
You said `QuizResponse` model is already being used in `get_quiz` no?
So created another class for use in `create_quiz`?
Something like `CreatedQuizResponse` or `QuizCreationResponse`? Nothing's stopping you from creating as many model classes as you want!